src/test/fakes.rs

```diff
@@ -111,6 +111,13 @@ impl<'a> FakeRelease<'a> {
         self
     }
 
+    pub(crate) fn platform<S: Into<String>>(mut self, platform: S) -> Self {
```

Comment:
```suggestion
    pub(crate) fn add_platform<S: Into<String>>(mut self, platform: S) -> Self {
```

This code adds a new target for the given platform, so it seems more appropriate to name it `add_platform`.